src/Test/Unit/Shell/ShellTest.php

```diff
@@ -188,10 +187,10 @@ public function testExecuteException()
             ->method('sanitize')
             ->with($this->stringContains('ls -al --password="123"'))
             ->willReturn('Command ls -al --password="***" failed');
-        $this->loggerMock->expects($this->once())
+        $this->loggerMock->expects($this->never())
             ->method('info')
             ->with($command);
```

Comment:
Please move `->with($command)` from `info` method mock to `debug` method  mock.
And you can remove `info`  method mock at all as it never used in the `Shell` class.

Reply:
Great point, thanks for catching this.